spec/acceptance/1_default_parameters_spec.rb

```diff
@@ -1,50 +1,38 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 
 # It is sometimes faster to host confluence / java files on a local webserver.
 # Set environment variable download_url to use local webserver
-# export download_url = 'http://10.0.0.XXX/'
-download_url = ENV['download_url'] if ENV['download_url']
-download_url = if ENV['download_url']
-                 ENV['download_url']
-               else
-                 'undef'
-               end
-java_url = if download_url == 'undef'
-             'http://download.oracle.com/otn-pub/java/jdk/7u71-b14/'
-           else
-             download_url
-           end
+# export BEAKER_FACTER_DOWNLOAD_URL = 'http://10.0.0.XXX/'
 
-describe 'confluence', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+describe 'confluence' do
   it 'installs with defaults' do
     pp = <<-EOS
-      $jh = $osfamily ? {
-        default   => '/opt/java',
-      }
-      if versioncmp($::puppetversion,'3.6.1') >= 0 {
-        $allow_virtual_packages = hiera('allow_virtual_packages',false)
-        Package {
-          allow_virtual => $allow_virtual_packages,
-        }
-      }
-      deploy::file { 'jdk-7u71-linux-x64.tar.gz':
-        target          => $jh,
-        fetch_options   => '-q -c --header "Cookie: oraclelicense=accept-securebackup-cookie"',
-        url             => #{java_url},
-        download_timout => 1800,
-        strip           => true,
+      $jh = '/opt/java'
+      file { $jh:
+        ensure => directory,
+      } ->
+      archive { 'jdk11':
+        path            => '/tmp/zulu8.66.0.15-ca-jdk8.0.352-linux_x64.tar.gz',
+        source          => pick(fact('download_url'), 'https://cdn.azul.com/zulu/bin/zulu8.66.0.15-ca-jdk8.0.352-linux_x64.tar.gz'),
+        extract         => true,
+        extract_command => 'tar xfz %s --strip-components=1',
+        extract_path    => $jh,
+        creates         => "${jh}/bin/java",
+        cleanup         => true,
       } ->
       class { 'confluence':
-        version             => '5.5.6',
-        download_url        => #{download_url},
-        javahome            => $jh,
+        version      => '5.5.6',
+        jvm_type     => 'oracle-jdk-1.8',
```

Comment:
Added this because jdk11 is the default but confluence 5.5.6 does not run on java11 ...